internal/util/util.go

```diff
@@ -22,3 +29,35 @@ func LookupImage(ctx *types.SystemContext, store storage.Store, image string) (*
 	}
 	return localImage, nil
 }
+
+// ExportFromReader reads bytes from given reader and exports to external tar, directory or stdout.
+func ExportFromReader(input io.Reader, isDir bool, isStdout bool, path string) error {
```

Comment:
Couldn’t this just accept a `BuildOutputOption` argument, simplifying the only caller, and completely eliminate the risk of mismatching the order of the bool options?